lib/aliyun/oss/bucket.rb

```diff
@@ -592,7 +592,7 @@ def bucket_url
       # @param [String] http_method URL的有效时间，单位为秒，默认为60s
       # @param [String] content_type URL的有效时间，单位为秒，默认为60s
       # @return [String] 用于直接访问Object的URL
-      def object_url(key, sign = true, expiry = 60, http_method: "GET", content_type: nil)
+      def object_url(key, sign = true, expiry = 1800, http_method: "GET", content_type: nil)
```

Comment:
Out of curiosity why you can't simply apply this in storage manager?
https://github.com/AcornsGrowAU/backend-ind/blob/master/app/services/invest_processing/storage_manager.rb#L16-L20

Reply:
its not for the reports, its for `CarrierWave`. https://github.com/bosen/carrierwave-aliyun/commit/7c099d5c858b6cba9f738bd1a017d5447bba7533#diff-f7d38ec9bc4db6925074df14e0a96047L135